### PR TITLE
adding support for ordered_from, brand_name, site_* top level fields, and *_language fields in browser and app complex fields

### PR DESCRIPTION
**Pull request by rsomavarapu-sift**
**Purpose**
Add additional fields to support user-level and order/transaction-level segmentation of the data which can be used for various purposes in the future like reporting, workflow management, permissions, etc.

**Technical Overview**
- Add support for `$client_language` field to `$app` complex field
- Add support for `$accept_language` and `content_language` fields to `$browser` complex field
- Add support for `$brand_name`, `$site_country` and `$site_domain` fields to custom events and all reserved events except `$chargeback`, `$link_session_to_user` and `$flag_content`
- Add support for `$ordered_from` complex field to `$create_order`, `$update_order`, and `$transaction` events

**Testing Plan**
- Updated unit tests
- Send sample events using SiftClient to the internal experiment environment and verify the changes using the internal experiment console.

**Deployment**
- Publish the package using NuGet

### CODE REVIEW

File: Test/Test.cs [hunk 1]
```diff
@@ -129,32 +129,51 @@ public void TestEventRequest()
...
+                    }
+                },
+                site_country = "US",
+                site_domain = "sift.com",
+                brand_name = "sift"
             };
 
             // Augment with custom fields
             createOrder.AddField("foo", "bar");
+            Console.WriteLine(createOrder.ToJson());
```
Comment (jkbach) [inline Test/Test.cs]: nit: please remove

File: Sift/Schema/ComplexTypes/browser.json [hunk 2]
```diff
@@ -5,6 +5,12 @@
   "properties": {
     "$user_agent": {
       "type": [ "string", "null" ]
+    },
+    "$accept_language": {
```
Comment (jkbach) [inline Sift/Schema/ComplexTypes/browser.json]: If you're feeling brave, could be a good chance to write tests for `browser` as well.

File: Test/Test.cs (hunk 1 above)
Comment (rsomavarapu-sift) [reply Test/Test.cs]: good catch. removing it.

**Comment by rsomavarapu-sift** [comment]
@jkbach good point. as far as I know, this library is currently in beta and we definitely have to add more functionality to it but I am not sure about the timeline.

File: Sift/Schema/ComplexTypes/browser.json (hunk 2 above)
Comment (rsomavarapu-sift) [reply Sift/Schema/ComplexTypes/browser.json]: done

**Pull request merged**